drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -747,17 +759,21 @@ function DragAndDropBlock(runtime, element, configuration) {
 
         $.post(url, JSON.stringify(data), 'json')
             .done(function(data){
-                state.last_action_correct = data.correct;
-                if (data.correct) {
-                    state.items[item_id].correct = true;
-                    state.items[item_id].submitting_location = false;
-                } else {
-                    delete state.items[item_id];
-                }
-                state.feedback = data.feedback;
-                if (data.finished) {
-                    state.finished = true;
-                    state.overall_feedback = data.overall_feedback;
+                state.items[item_id].submitting_location = false;
+                // In standard mode we immediately return item to the bank if dropped on wrong zone.
+                // In assesment mode we leave it at the dropped place until explicit answer submission.
```

Comment:
@itsjeyd It's not just you, that was a weird sentence indeed. Fixed with your suggestion.